hpe3par_sdk/client.py

```diff
@@ -3962,3 +3962,31 @@ def remoteCopyGroupStatusStoppedCheck(self, remote_copy_group_name):
         :              state
         """
         return self.client.remoteCopyGroupStatusStoppedCheck(remote_copy_group_name)
+
+    def isScheduleActive(self, schedule_name):
```

Comment:
@ansarars How is this helping in reducing code duplicacy? You have simply copied the client changes here in sdk which is not we want.
Please add a getScheduleStatus method in client which returns either active or suspended status and in sdk we will have only one method isScheduleActive which will return True if client returns active status else False(which means schedule status is suspended)
Also, in getScheduleStatus method is client should raise an SSHException if none of the status active/suspended is found from the result of getSchedule().

In ansible, we will simply call the isScheduleActive() of SDK and check whether schedule is active or not.